### ISSUE TITLE
How to open the brunch configuration menu

### ISSUE THREAD

**Issue by DangJiYi** [sebanc/brunch#1509]
where is this "ChromeOS (settings)"?
![Screenshot 2022-04-12 10 13 18 PM](https://user-images.githubusercontent.com/53961226/162983059-e56150e5-c1a9-42a7-af64-ab212bdef860.png)

Why is my interface like this
![Screenshot 2022-04-12 10 20 43 PM](https://user-images.githubusercontent.com/53961226/162984031-21a991d9-a90b-41cd-a91e-573d5ff7f464.png)

**Comment by KarlS0808** [comment]
You can also go into it by doing ctrl+alt+f2 and logging in as root then doing edit-grub-config or edit-brunch-config

**Comment by DangJiYi** [comment]
> You can also go into it by doing ctrl+alt+f2 and logging in as root then doing edit-grub-config or edit-brunch-config

No, my interface is still the old version. I don't know how to open the new configuration menu

**Comment by tejasraman** [comment]
Update brunch using PWA. See #1512
